packages/amqp/test/utils/testContext.ts

```diff
@@ -208,6 +213,7 @@ export interface Dependencies {
 
   // vendor-specific dependencies
   transactionObservabilityManager: TransactionObservabilityManager
+  messageMetricsManager?: MessageMetricsManager
```

Comment:
can you also add documentation about it to readme file?

Reply:
added both JSDoc comments and section in README. But I'm not sure if JSDoc comments wouldn't be enough as user only needs to know when the methods are executed and what are the parameters